cheroot/ssl/builtin.py

```diff
@@ -43,20 +43,25 @@ class BuiltinSSLAdapter(Adapter):
     """The ssl.SSLContext that will be used to wrap sockets where available
     (on Python > 2.7.9 / 3.3)
     """
+    ciphers = None
+    """The ciphers list of SSL."""
```

Comment:
The active PEP257 still [has attribute docstrings mentions](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring)